shell/platform/darwin/ios/framework/Source/FlutterAppDelegateTest.mm

```diff
@@ -67,8 +67,9 @@ - (void)testLaunchUrl {
                             openURL:[NSURL URLWithString:@"http://myApp/custom/route?query=test"]
                             options:@{}];
   XCTAssertTrue(result);
-  OCMVerify([self.mockNavigationChannel invokeMethod:@"pushRouteInformation"
-                                           arguments:@{@"location" : @"/custom/route?query=test"}]);
+  OCMVerify([self.mockNavigationChannel
+      invokeMethod:@"pushRouteInformation"
+         arguments:@{@"location" : @"http://myApp/custom/route?query=test"}]);
```

Comment:
Isn't it weird to be using the "http" protocol for this?

Reply:
An [universal link](https://developer.apple.com/documentation/xcode/allowing-apps-and-websites-to-link-to-your-content?language=objc) is either a http or https protocol.

Reply:
This is the framework pr https://github.com/flutter/flutter/pull/119968, it just surface whatever url that is sent here to the developer in [RouteInformation.url](https://master-api.flutter.dev/flutter/widgets/RouteInformation/uri.html)

The goal is to let developer to be able to handle differently based on what uri to launch the app, for example they may want to block certain route if the app is launch from `example.com` but not when it is launch from `another-example.com`